sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKind.java

```diff
@@ -10,7 +10,13 @@
     /**
      * The Partition of a item is calculated based on the hash value of the PartitionKey.
      */
-    HASH("Hash");
+    HASH("Hash"),
+
+    RANGE("Range"),
+    /**
+     * The Partition of a item is calculated based on the hash value of multiple PartitionKeys.
+     */
+    MULTI_HASH("MultiHash");
```

Comment:
Naming suggestions. 

Aligning the name with the public branding would be ideal (MultiHash, SubParititoning, MultiPartHash)? (Its a REST contract, we can at-least change the POJO type name).

Reply:
MultiHash has been used for V3 too. Maintained the same here.